### DISCUSSION TITLE
[Deploy] Is it possible to disable the smart deploy using mlp on a project?

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#260] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11.0.2

### Description

Hi,
we use `mlp` as deploy tool, to deploy our projects on k8s.

Is there a way to disable, for a specific project, the "_Smart Deploy_" feature?

I didn't found that on the documentation.

Thanks.

**Comment by davidebianchi** [comment]
Hi! Why do you need to disable the smart deploy?

**Reply by frengooo** [reply]
A client asks if it's possible at the moment, to force the developers in the initial phase of a project to always release the latest version of the services.

**Reply by davidebianchi** [reply]
Ok, clear. So do you need to release the non semver tag by default instead of remove smart deploy, correct?

**Reply by frengooo** [reply]
Probably it could be a solution if isn't possible to remove the "_Smart Deploy_".

**Reply by davidebianchi** [reply]
We are thinking to set the `Smart Deploy` feature as the standard feature, since it is how Kubernetes works by default. I think it should be a straightforward way to manage the deploy. Why do you need specifically to remove it?

**Reply by frengooo** [reply]
I agree with the `Smart Deploy` used by default, I think it's a great feature.
Despite that, there are clients that need to disable it for several reasons:
- initial development phase &rarr; I want to always release the latest version without letting the user choose for smart deployment;
- release flow &rarr; I test a specific tag of my project configuration and want to release the whole tag, with all the services inside (a brand new release of the whole project).

We discussed the use cases above with different clients, so I think this feature should be useful.

**Reply by frengooo** [reply, marked as answer]
We don't need this anymore.

Anyway, at the moment, is not possible to "permanently" remove the Smart Deploy toggle to force all users to always perform a "Deploy All".
The only solution (a little bad in my opinion) is to force the full deploy directly in the pipelines, but it's a little misleading from the users perspective.
